### ISSUE TITLE
Accessibility bug squashing workflow discovery

### ISSUE THREAD

**Issue by cvalarida** [department-of-veterans-affairs/va.gov-team#13610]
## Problem Statement
Accessibility specialists and engineers have a difficult time closing accessibility bugs. This ticket is to capture the discovery work to figure out _why_ this is as much of a struggle as it is.

---
## Tasks
- [ ] Diagram the workflow to discover bottlenecks in the process
- [ ] Brainstorm ways to fix the bottlenecks
- [ ] Write tickets for the workflow improvements

**Comment by cvalarida** [comment]
I met with @1Copenut and @bkjohnson to discuss the a11y bug squashing workflow from start to finish. There were parts of the lifecycle we didn't dive deep into

# A Bug's Life
**Note:** The following diagrams aren't intended to be a 100% faithful representation of these workflows. Each bug is a little different, and our sample set of bugs when talking about the workflow was pretty small at the time of discovery. In particular, these diagrams don't capture things like bug fixes introducing regressions in other behavior.

The originals for the following diagrams can be found [on Lucid Chart](https://app.lucidchart.com/invitations/accept/33705ab8-6e2a-4cd2-8a8c-cac8356ac954).

## Overview
![Overview](https://user-images.githubusercontent.com/12970166/93361187-6f090180-f7f9-11ea-980a-2409713c93c4.png)

## Determining the correct solution
![Experimentation](https://user-images.githubusercontent.com/12970166/93361183-6dd7d480-f7f9-11ea-9466-e89fc65bb93a.png)

## Implementing the solution
![Coding](https://user-images.githubusercontent.com/12970166/93361185-6e706b00-f7f9-11ea-83b1-7e1fb899a484.png)

# Takeaways
While talking to @1Copenut, the biggest bottleneck we could think of was around testing a solution that we we had low confidence in. For example, we weren't sure what would get iOS to read a heading text change, so we tried a few things, but for each code change, @1Copenut had to pull the branch, run the watch task, wait for it to finish, navigate to the right page, plug the data into the form until he could reproduce the bug.

A possible solution we came up with to shorten the feedback loop on changes we weren't confident in can be found in [this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/13436).

**Comment by cvalarida** [comment]
@1Copenut and I talked today and discussed some improvements we can make to the process.
- A11y tickets must have clear acceptance criteria which engineers can test against
- When this is not feasible, we need a discovery ticket in which one or more engineers work with the accessibility specialist to prototype a solution to create the acceptance criteria
  - If the experiments are simple enough to implement in the code base, that's an option too, of course
- The `Possible fixes (optional)` section in the a11y bug ticket will be renamed to indicate that this is the proposed solution

We also talked about what an artifact handoff from a11y specialists to engineering might look like, but didn't come to a firm conclusion on that one. CodePen was one possibility, but the effort there can be pretty high on the specialist's part.
